index.js

```diff
@@ -2,8 +2,13 @@ const Seneca = require('seneca')
 const GetPoll = require('./lib/get_poll_msg')
 const OpenPoll = require('./lib/open_poll_msg')
 const CastVote = require('./lib/cast_vote_msg')
+const Shapes = require('./lib/shapes')
+
+
+module.exports = async function (plugin_opts = {}) {
+  await Shapes.validatePluginOptions(plugin_opts)
```

Comment:
use module.exports.defaults = { ... }  for options validation - it can take Joi operations
https://github.com/rjrodger/optioner

Reply:
@rjrodger Is there any way to pass options to the Optioner module, instantiated by Seneca? I don't seem to be able to find a way to do so in the source code. I would like to pass the `{ allow_unknown: false }` option to Optioner in order to help users of the plugin easily catch typos in option names.